### PR TITLE
Add support for `playerctl` in the `now_playing` segment

### PR DESCRIPTION
**Pull request by IngoMeyer441**

### CODE REVIEW

File: segments/now_playing.sh [hunk 1]
```diff
@@ -121,6 +125,9 @@ __process_settings() {
 	if [ -z "$TMUX_POWERLINE_SEG_NOW_PLAYING_NOTE_CHAR" ]; then
 		export TMUX_POWERLINE_SEG_NOW_PLAYING_NOTE_CHAR="${TMUX_POWERLINE_SEG_NOW_PLAYING_NOTE_CHAR_DEFAULT}"
 	fi
+	if [ -z "$TMUX_POWERLINE_SEG_NOW_PLAYING_PLAYERCTL_FORMAT" ]; then
+		export TMUX_POWERLINE_SEG_NOW_PLAYING_PLAYERCTL_FORMAT="${TMUX_POWERLINE_SEG_NOW_PLAYING_PLAYERCTL_FORMAT}"
```
Comment (erikw) [inline segments/now_playing.sh:129]: The value should be `TMUX_POWERLINE_SEG_NOW_PLAYING_PLAYERCTL_FORMAT_DEFAULT` right?

I think it's the same issue on `TMUX_POWERLINE_SEG_NOW_PLAYING_RHYTHMBOX_FORMAT` below as well. Maybe you can fix it too? :)

**Review by erikw — CHANGES_REQUESTED** [review]

**Comment by erikw** [comment]
Thank you @IngoMeyer441. This PR looks good, just one comment to address :)

File: segments/now_playing.sh (hunk 1 above)
Comment (IngoMeyer441) [reply segments/now_playing.sh:129]: Ah, yes this is a copy-and-paste mistake :innocent: 
Both is fixed in the latest commit.

**Comment by erikw** [comment]
Thank you!

**Pull request merged**